EmailPiece.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:core';
+import 'package:enough_mail/enough_mail.dart';
+import 'package:intl/intl.dart';
+import 'package:summer2022/models/Digest.dart';
+
+class EmailPiece {
+  String EmailPieceID; // Unique ID of Image
+  String EmailID; // Unique ID of Email - made from Email contents hashed
+  DateTime EmailPieceTimestamp;
+  String EmailPieceSender;
+  String EmailPieceImageText;

```

Comment:
May want to exclude the class name from appearing in each of the property names for simplicity.